website/docs/d/org_vdc_nsxt_network_profile.html.markdown

```diff
@@ -0,0 +1,44 @@
+---
+layout: "vcd"
+page_title: "VMware Cloud Director: vcd_org_vdc_nsxt_network_profile"
+sidebar_current: "docs-vcd-datasource-nsxt-segment-profile-template"
+description: |-
+  Provides a data source to read Network Profile for NSX-T VDCs.
+---
+
+# vcd\_org\_vdc\_nsxt\_network\_profile
+
+Provides a data source to read Network Profile for NSX-T VDCs.
+
+Supported in provider *v3.11+* and VCD 10.4.0+ with NSX-T.
+
+## Example Usage
+
+```hcl
+data "vcd_org_vdc_nsxt_network_profile" "nsxt" {
+  org = "my-org"
+  vdc = "my-vdc"
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `org` - (Optional, but required if not set at provider level) Org name 
+* `vdc` - (Optional, but required if not set at provider level) VDC name 
```

Comment:
(Same as in above comment). Let's keep this part consistent with other pages, and let's keep the part in parenthesis canonical - without comments.

Reply:
Fixed